dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
The Stanford HARDI data set is under PDDL. That's what the librarians at
Stanford recommended.

On Thu, Aug 6, 2015 at 10:08 AM, Ariel Rokem <arokem@gmail.com> wrote:

> PDDL is supposedly the best for this purpose:
>
> http://opendatacommons.org/licenses/pddl/
>
> But CC0 is very similar:
>
> https://creativecommons.org/about/cc0
>
> On Thu, Aug 6, 2015 at 10:00 AM, Eleftherios Garyfallidis <
> notifications@github.com> wrote:
>
>> In dipy/data/fetcher.py
>> <https://github.com/nipy/dipy/pull/681#discussion_r36438492>:
>>
>> >      """
>> > -    folder = pjoin(dipy_home, 'mni_template')
>> > +    files, folder = fetch_mni_template()
>> >      file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
>>
>> Can you sennd me a link about the PDDL license? Which license would be
>> the best? A CC one?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/nipy/dipy/pull/681/files#r36438492>.
>>
>
>